src/components/providers/authentication-provider.tsx

```diff
@@ -5,7 +5,7 @@ import Keycloak from "keycloak-js";
 import { useCallback, useEffect } from "react";
 
 interface Props {
-  children: JSX.Element;
+  children: JSX.Element[];
```

Comment:
What does this array notation mean or change? Not that I know, just curious. Just seeing JSX.Element, ReactElement and ReactNode used interchangably.

Reply:
Good question, chatGpt provided me with an informative answer!

Reply:
Juuso came across with this solution. There was an error for multiple children, and I coudn't resolve it.